python-sdk/src/astro/databases/snowflake.py

```diff
@@ -552,7 +552,9 @@ def create_table_using_schema_autodetection(
         # Since this method is used by both native and pandas path we cannot skip this step.
         self.truncate_table(table)
 
-    def is_native_load_file_available(self, source_file: File, target_table: BaseTable) -> bool:
+    def is_native_load_file_available(
```

Comment:
@pankajastro maybe we should change this to static? in base class too?

Reply:
isn't this is also an override of the base class method?